src/services/notifications.service.js

```diff
@@ -323,7 +327,10 @@ const initialiseNotificationsStatus = (emailsToSend) => {
   for (let index in emailsToSend) {
     notifications.push({
       time: undefined,
-      sent: false,
+      sent:
+        emailsToSend.length > 1 && emailsToSend[index].type === "RNG_PENDING" // In order to prevent RN pending emails from being sent to FBO when RNG service has been resolved
```

Comment:
does this mean there will be a notification marked as sent when it actually wasn't sent but just isn't needed?

Reply:
As discussed, this is fine.